Formula/ocaml-zarith.rb

```diff
@@ -0,0 +1,42 @@
+class OcamlZarith < Formula
+  desc "OCaml library for arbitrary-precision arithmetic"
+  homepage "https://github.com/ocaml/Zarith"
+  url "https://github.com/ocaml/Zarith/archive/release-1.11.tar.gz"
+  sha256 "f996af120a10fd06a8272ae99b7affd57cef49c57a3a596e2f589147dd183684"
+  license "LGPL-2.0-only"
+
+  depends_on "ocaml-findlib" => :build
+  depends_on "gmp"
+  depends_on "ocaml"
+
+  def install
+    ENV["OCAMLFIND_DESTDIR"] = lib/"ocaml"
+
+    (lib/"ocaml").mkpath
+    cp Formula["ocaml"].opt_lib/"ocaml/Makefile.config", lib/"ocaml"
+
+    # install in #{lib}/ocaml not #{HOMEBREW_PREFIX}/lib/ocaml
+    inreplace lib/"ocaml/Makefile.config" do |s|
+      s.change_make_var! "prefix", prefix
+    end
+
+    ENV.deparallelize
```

Comment:
Did you check if the build could succeed without this line?

Reply:
Yes, it fails without this (sadly). There's a generated file missing as a dependency in the Makefile.